submissions/merowing/js-post-oop/index.js

```diff
@@ -0,0 +1,76 @@
+import { print } from './js/lib.js';
+
+class Creature {
+   constructor(name, gender, legs, say) {
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.say = say;
+   }
+
+   message() {
+      return `${this.species}; <strong>${this.name}</strong>; ${this.gender}; ${this.hands}; ${this.legs}; <em>${this.say}</em>;${this.friends}`;
```

Comment:
Class shouldn't handle properties it doesn't own.
Base class never knows what properties child classes intorduce on top of shared ones.
That's the whole point of encapsulation in OOP.

Also, why name, gender, legs and say are defined in the base class while species and hands are not?
What was the rule you made decisions on this?

Reply:
Removed message from main class.
Rewritten Child classes.

Reply:
Not the right direction.
> Class shouldn't handle properties it doesn't own.
This is still valid. And applicable to any class - base or child.

Reply:
done